src/dirs.rs

```diff
@@ -50,3 +50,32 @@ pub fn bin_dir() -> PathBuf {
             d
         })
 }
+
+#[macro_export]
+macro_rules! ensure_dir {
```

Comment:
does this need to be a macro?

Reply:
Doing otherwise would be rather inconvenient. It’d mean having a separate ensure method for each of the config first.